draft-js-side-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable react/no-array-index-key */
 import React from 'react';
+import PropTypes from 'prop-types';
 import DraftOffsetKey from 'draft-js/lib/DraftOffsetKey';
+import BlockTypeSelect from '../BlockTypeSelect';
 
-export default class Toolbar extends React.Component {
+class Toolbar extends React.Component {
 
```

Comment:
add default `children` prop with all current block buttons

Reply:
added renderDefaultButtons inCase there is no this.props.children